assets/pages/paper-subscription-landing/components/form.jsx

```diff
@@ -5,45 +5,91 @@
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
-import { type PaperBillingPlan } from 'helpers/subscriptions';
+import { type Option } from 'helpers/types/option';
+import { getNewsstandSaving, getNewsstandPrice, type PaperBillingPlan } from 'helpers/subscriptions';
 import { type Action } from 'components/productPage/productPagePlanForm/productPagePlanFormActions';
 import ProductPagePlanForm, { type StatePropTypes, type DispatchPropTypes } from 'components/productPage/productPagePlanForm/productPagePlanForm';
 
 import { type State } from '../paperSubscriptionLandingPageReducer';
 import { setPlan } from '../paperSubscriptionLandingPageActions';
 
 
+// ---- Helpers ----- //
+
+const getPriceStr = (price: Option<number>): Option<string> =>
+  (price ? `From £${price} per month` : null);
+
+const getSavingStr = (subscription: Option<number>, newsstand: Option<number>): Option<string> =>
```

Comment:
Can this function be generalised for other products? If so then move it to subscriptions.js

Reply:
it's ab it of a one-off for now i'd advocate for generalising it when it's actually reused